sequoia/settings/offline_rl/setting.py

```diff
@@ -0,0 +1,40 @@
+import d3rlpy
+from sklearn.model_selection import train_test_split
+from typing import ClassVar, List, Tuple, Dict
+
+
+from sequoia.settings.base import Setting
+from torch.utils.data import DataLoader
+from sequoia.settings.base import Method
+from dataclasses import dataclass
+
+from simple_parsing.helpers import choice
+
+
+@dataclass
+class OfflineRLSetting(Setting):
+    available_datasets: ClassVar[list] = ["cartpole-replay",  # Cartpole Replay
```

Comment:
Can we add more datasets, if the other dependencies (atari-py for example) are installed?

Reply:
Yes! I'm running out of space on my computer at the moment, but I will try to test as many datasets as I can